examples/screen-grid/app.js

```diff
@@ -54,7 +54,6 @@ class Root extends Component {
       <MapGL
         {...viewport}
         mapStyle="mapbox://styles/mapbox/dark-v9"
-        perspectiveEnabled={true}
```

Comment:
It's a bit unfortunate to remove them here in examples, even if they are duplicated.